cloudbuild.yaml

```diff
@@ -0,0 +1,23 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+
+# this must be specified in seconds. If omitted, defaults to 600s (10 mins)
+timeout: 1200s
+# this prevents errors if you don't use both _GIT_TAG and _PULL_BASE_REF,
+# or any new substitutions added in the future.
+options:
+  substitution_option: ALLOW_LOOSE
+steps:
+  - name: gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20220609-2e4c91eb7e
+    entrypoint: make
+    env:
+    - REGISTRY=gcr.io/k8s-staging-cloud-provider-os
```

Comment:
AFAICT this is the correct registry, defined here: https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/manifests/k8s-staging-cloud-provider-os/promoter-manifest.yaml